modules/migration-mtc-release-notes-1-7.adoc

```diff
@@ -0,0 +1,27 @@
+
+// Module included in the following assemblies:
+//
+// * migration_toolkit_for_containers/mtc-release-notes.adoc
+:_content-type: REFERENCE
+[id="migration-mtc-release-notes-1-7_{context}"]
+= {mtc-full} 1.7 release notes
+
```

Comment:
You need some introductory text between the H1 and H2.

Reply:
Will not fix. I hope this isn't a blocker.

Reply:
@RichardHoch, please consider opening a follow-up PR to add this heading. It's a mod docs and Jupiter requirement that's not quite a blocker and not quite enough for a bug.